src/components/tables/cells/EntityName.tsx

```diff
@@ -9,35 +9,19 @@ function EntityName({ name }: Props) {
     return (
         <TableCell
             sx={{
-                'minWidth': 256,
-                '& > *': {
-                    borderBottom: 'unset',
-                },
+                minWidth: 256,
             }}
         >
-            <Tooltip title={name} placement="bottom-start">
-                <Box>
-                    <span
-                        style={{
-                            height: 16,
-                            width: 16,
-                            backgroundColor:
-                                getDeploymentStatusHexCode('ACTIVE'),
-                            borderRadius: 50,
-                            display: 'inline-block',
-                            verticalAlign: 'middle',
-                            marginRight: 12,
-                        }}
-                    />
-                    <span
-                        style={{
-                            verticalAlign: 'middle',
-                        }}
-                    >
-                        {name}
-                    </span>
-                </Box>
-            </Tooltip>
+            <>
+                <EntityStatus />
```

Comment:
Broke the entitystatus into a component and do not need tooltip as we always show the full entity name.